### PR TITLE
CAPI: Support Marker feature for CAPI.

### PR DESCRIPTION
**Pull request by JSUYA**
Add lottie_animation_get_markerlist() api
User can use this api and get information about each marker.

### CODE REVIEW

File: src/binding/c/lottieanimation_capi.cpp [hunk 1]
```diff
@@ -205,4 +205,28 @@ lottie_animation_property_override(Lottie_Animation_S *animation,
...
+{
+   if (!animation) return nullptr;
+
+   LOTMarkerList *markerlist;
+   auto markers = animation->mAnimation->markers();
+   if (markers.size() == 0) {
+       return nullptr;
+   }
+
+   markerlist = new LOTMarkerList();
```
Comment (smohantty) [inline src/binding/c/lottieanimation_capi.cpp]: who will free this allocation ?

File: src/binding/c/lottieanimation_capi.cpp [hunk 2]
```diff
@@ -205,4 +205,28 @@ lottie_animation_property_override(Lottie_Animation_S *animation,
...
+   markerlist->size = 0;
+   markerlist->ptr = new LOTMarker[markers.size()]();
+
+   for(size_t i = 0; i < markers.size(); i++) {
+       markerlist->ptr[i].name= std::move(std::get<0>(markers[i]).c_str());
```
Comment (smohantty) [inline src/binding/c/lottieanimation_capi.cpp]: std::move will not help here. as we are directly accessing the underline pointer.

**Comment by JSUYA** [comment]
@smohantty 
I also think returns markerlist is wrong.
so I updated patch. please review again :)

**Comment by JSUYA** [comment]
I updated commit. please review again

**Pull request merged**